docs/personas.md

```diff
@@ -0,0 +1,31 @@
+# Personas
+## [P1] <a name="P1">Propietario</a>
+Dueño y CEO del restaurante. Como propietario es el encargado de tomar decisiones acertadas para el futuro de su negocio, además de gestionar el personal. 
+
+## [P2] <a name="P2">Camarero</a>
+Un empleado del restaurante. Encargado de atender de forma a las mesas, servir la comida hecha por el [cocinero](#P3) y del trato con los [clientes](#P4).
+
+## [P3] <a name="P3">Cocinero</a>
+Empleado del restaurante encargado de hacer las comidas que piden los [clientes](#P4).
+
+## [P4] <a name="P4">Cliente</a>
+Consumidor que haya pasado por el restaurante. Se encarga de registrar una opinión nueva en el sistema.
```

Comment:
Debéis entender bien el concepto de la nube, que implica, entre otras cosas, "multi-tenancy", es decir, que no se hace la aplicación para "un" negocio, sino para todos los negocios. Para un negocio basta con que se haga una app o aplicación de escritorio.

Reply:
Mi idea inicial es diseñar la plataforma y que todo restaurante que lo desee la pueda implantar, ¿dónde di a entender lo contrario?

Reply:
¿Dónde dices que el valor añadido de esa aplicación está en que se ejecute en la nube?